chore: add allowlist requirement in email alert (#1452)
WalletConnect/walletconnect-docs: docs/web3modal/shared/email-alert.mdx

:::caution
Email Login is currently in Beta. \

> In order to access this feature, your Dapp must be on [allowlist](/cloud/relay#allowlist).

Reviewer: ```suggestion
In order to use this feature you must also configure the [allowlist](/cloud/relay#allowlist) for your app.
```
